sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/AppendBlobClient.java

```diff
@@ -53,6 +55,7 @@
  */
 @ServiceClient(builder = SpecializedBlobClientBuilder.class)
 public final class AppendBlobClient extends BlobClientBase {
+    private final ClientLogger logger = new ClientLogger(AppendBlobClient.class);
```

Comment:
this should be static . Please don't create non-static logger instances going forward. There's going to be checkstyle rule enforcing that in the future.

Reply:
I think she started work on this before that change was merged into main, so the old pattern was likely still in place. Rabab, you can check the current main branch for examples on what this will look like, but essentially add the static keyword and change the name of the field to be all caps.